### ISSUE TITLE
Learner gets 500 server error when the class has a visible lesson

### ISSUE THREAD

**Issue by radinamatic** [learningequality/kolibri#6169]
### Observed behavior

No problem when the class has an opened/started quiz, but as soon as a lesson is made visible, learner gets the 500 error (_Something went wrong_) on **Learn > Classes** page:

![learner-cant-see-classes](https://user-images.githubusercontent.com/1457929/69838460-ef06b080-1253-11ea-8af6-a0941c6f1859.gif)

Seems there is also some weird learner logout there in the middle, as `id: "NOT_AUTHENTICATED", metadata: {view: "Learner Classroom Viewset List"` is visible in the request preview.

As soon as the lesson is made invisible again, and the learner signs in again, they can see the **Learn > Classes** page no prob:

![learner-cant-see-classes-with-lessons](https://user-images.githubusercontent.com/1457929/69838911-c97aa680-1255-11ea-8802-6b47878b20b5.gif)

### Expected behavior
No errors and working lesson.

### User-facing consequences
Inability to access their class and the assigned resources.

### Errors and logs
(see screencast)

### Steps to reproduce
Sign in as a learner when your class has a started lesson.

### Context
• Kolibri version: kolibri_0.13.0b3-0ubuntu1
• Operating system: Ubuntu 16.04
• Browser: Firefox & Chrome

**Comment by mrpau-julius** [comment]
Same error on Android devices version 4.4 and 6.0
![Screenshot_20191129-154242](https://user-images.githubusercontent.com/38421180/69852759-11361800-12c0-11ea-9d42-9fd5058b151c.png)

 ### error details 
```
{
  "request": {
    "path": "/learn/api/learnerclassroom/",
    "params": {
      "1575013340098": 1575013340098
    },
    "headers": {
      "Content-Type": "application/json",
      "Accept": "application/json, application/json;q=0.8, text/plain;q=0.5, */*;q=0.2",
      "X-CSRFToken": "S0iy9P1qj6GS5pyHLuFu31tK3yHFvGadeMRpjxi2084hiaU8MP8m2zK0OM7mhg2h"
    },
    "method": "GET",
    "canceled": false
  },
  "url": "/learn/api/learnerclassroom/?1575013340098=1575013340098",
  "raw": {},
  "status": {
    "code": 500,
    "text": "Internal Server Error"
  },
  "headers": {
    "Date": "Fri, 29 Nov 2019 07:42:21 GMT",
    "Vary": "Cookie",
    "Server": "0.0.0.0",
    "Content-Length": "27",
    "X-Frame-Options": "SAMEORIGIN",
    "Content-Type": "text/html"
  },
  "entity": "<h1>Server Error (500)</h1>"
}
```

**Comment by mrpau-richard** [comment]
I also reproduced this on Windows 7 server and Android 4 client.

**Comment by bjester** [comment]
May get resolved in next beta release by https://github.com/learningequality/kolibri/pull/6159

**Comment by rtibbles** [comment]
Should be fixed in #6159 - please reopen if not.
